src/sass/app/sell_popup.scss

```diff
@@ -95,6 +95,9 @@
                         }
                     }
                 }
+                #table_digits_container {
+                    margin-bottom: 20px;
```

Comment:
is it always 20px?

Reply:
Yeah, and it looks okay on both phone and desktop. Should I use `vh` instead?